src/gui/gui_msgs.h

```diff
@@ -37,45 +37,62 @@ constexpr char help_msg[] =
 
 List of common options:
 
-  --printconf          Print the location of the default configuration file.
+  --printconf              Print the location of the default configuration file.
 
-  --editconf           Open the default configuration file in a text editor.
+  --editconf               Open the default configuration file in a text editor.
 
-  -c <command>         Run the specified DOS command before running FILE.
-                       Multiple commands can be specified.
+  --eraseconf              Delete the configuration file.
```

Comment:
We need to specify which one (default or local).

Reply:
It does `const auto path = (get_platform_config_dir() / file).string();` so the default one.  I'll specify.

Reply:
Specified "default"